sdk/trace/src/main/java/io/opentelemetry/sdk/trace/ExceptionEventUtil.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.sdk.trace.data.EventData;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+
+final class ExceptionEventUtil {
+  private static final ExceptionEventFactory factory = initializeEventFactory();
+
+  static EventData create(Throwable exception, Attributes additionalAttributes, long epochNanos) {
+    return factory.create(exception, additionalAttributes, epochNanos);
+  }
+
+  private static ExceptionEventFactory initializeEventFactory() {
+    try {
+      Class<?> clazz =
```

Comment:
I worry a bit about having this reflection here, which will get logged as a warning on every SDK startup on a JDK 11+. I'm not sure I have a better solution, though.